api-reference/10 UI Widgets/dxHtmlEditor/3 Methods/getModule(moduleName).md

```diff
@@ -3,13 +3,15 @@ id: dxHtmlEditor.getModule(moduleName)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Gets instance of a [module](https://github.com/DevExpress/devextreme-quill/tree/master/modules). 
 
 ##### return: Object
-<!-- Description goes here -->
+A module's instance. 
 
 ##### param(moduleName): String
-<!-- Description goes here -->
+The name of a [registered](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#registercomponents) module. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can use DevExtreme Quill modules, DevExtreme UI [modules](https://github.com/DevExpress/DevExtreme/tree/20_2/js/ui/html_editor/modules) or create custom ones.
+
+Make sure that modules you want to get are not disabled in the [customizeModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/#customizeModules) function.
```

Comment:
Who does the disabling? The developer or the module can just be disabled by default?

Reply:
The developer, and the only place for this is the `customizeModules` function. Once the dev disabled the module, they can't get via this method.